flaml/searcher/flow2.py

```diff
@@ -306,10 +307,10 @@ def normalize(self, config) -> Dict:
                         # normalize categorical
                         if key in self._ordered_cat_hp:
                             l, d = self._ordered_cat_hp[key]
-                            config_norm[key] = d[value]/len(l)
+                            config_norm[key] = (d[value]+0.5)/len(l) # center
```

Comment:
why?

Reply:
because we use floor() to convert from float to int.